tests/components/deconz/test_light.py

```diff
@@ -268,48 +273,25 @@ async def test_disable_light_groups(hass):
         options={deconz.gateway.CONF_ALLOW_DECONZ_GROUPS: False},
         get_state_response=data,
     )
-    assert "light.rgb_light" in gateway.deconz_ids
-    assert "light.tunable_white_light" in gateway.deconz_ids
-    assert "light.light_group" not in gateway.deconz_ids
-    assert "light.empty_group" not in gateway.deconz_ids
-    assert "light.on_off_switch" not in gateway.deconz_ids
-    # 3 entities
-    assert len(hass.states.async_all()) == 5
-
-    rgb_light = hass.states.get("light.rgb_light")
-    assert rgb_light is not None
-
-    tunable_white_light = hass.states.get("light.tunable_white_light")
-    assert tunable_white_light is not None
-
-    light_group = hass.states.get("light.light_group")
-    assert light_group is None
 
-    empty_group = hass.states.get("light.empty_group")
-    assert empty_group is None
+    assert len(hass.states.async_all()) == 5
+    assert hass.states.get("light.rgb_light")
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)